Deliverable 4 david

7thStringofZhef opened the pull request:
I wrote comments for all my tests and wrote up and tested an email sender. Still have work to do, but I figure it'd be good to have a commented up version of code on the master branch.

Also just added my file receiving function.

AdityaMahadevan commented on skeleton-web-services/src/main/resources/static/js/main.js:
```diff
@@ -5913,6 +5890,10 @@ var m = require('mithril'),
     ProfileComponent = require('./components/Profile/Profile');
 
+function verify() {
+    //
```
verify() either needs to be implemented or removed?

AdityaMahadevan commented on skeleton-web-services/src/test/java/cs1530/osgfreesurfer/services/JobServiceTest.java:
```diff
@@ -36,6 +36,7 @@ public void tearDown()
 	}
 	
 	@Test
+	//Test service's find all function. There should be the initial 2 jobs
```
The comments fo all of these tests look good :+1:

7thStringofZhef commented:
Hmm...that's the frontend chunk I loaded in. Let me try pulling the newest frontend set from github and trying it again.

AdityaMahadevan commented on skeleton-web-services/target/classes/static/js/main.js:
```diff
@@ -5913,6 +5890,10 @@ var m = require('mithril'),
     ProfileComponent = require('./components/Profile/Profile');
 
+function verify() {
+    //
```
another empty verify() ?

AdityaMahadevan commented on skeleton-web-services/src/main/java/cs1530/osgfreesurfer/services/UserServiceBean.java:
```diff
@@ -85,4 +90,15 @@ public boolean checkIfLoggedOn(long id) {
 	}
 
+	@Override
+	public void emailUser(String email)
```
:+1: (assuming it works)

7thStringofZhef commented:
I found a problem with email and fixed it. I also added session support, but that forces people to install and run Redis, so I also left instructions on how to disable that function. 

I pulled the newest version of the frontend down and packaged it in, but if something's wrong with verify() or w/e, there's really nothing I can do about it. It's not my function, and I don't know the first thing about JS

AdityaMahadevan commented:
Looks like the new version of the frontend doesn't even have that file with the verify function anymore. Also, it sounds like using something like Redis might need approval? 

We can discuss during the next sprint. I'm going to merge this now.
